files/jargon.html

```diff
@@ -146,14 +146,10 @@
 
 ???
 
-Cloud systems physically resemble clusters or supercomputers, but
-with greater heterogeneity among components, and less coupling, often
-without an interconnect at all. Cloud servers are generally accessed
-by connecting to their services over the network. Multiple virtual
-machines may reside on individual physical machines. Typical tasks
-are driven by user connections from the outside to the various "front
-end" services (e.g. web sites) which pass traffic on to the "back
-end" servers (e.g. databases) to present results to users.
+A collection of standalone computers that are networked together. The will
```

Comment:
At the end of line 149 here, I think "The will" should be "They will"?

Reply:
Ok